webapp/plugins/plugins.js

```diff
@@ -57,17 +68,19 @@ export const loadPlugins = async config => {
   };
 
   // setup reducers decorators
-  // TODO: This needs to be generalized
   chainReducers = [];
   nodeReducers = [];
   walletsReducers = [];
-  pluginsReducers = [];
   reducersDecorators = {
     chainReducer: chainReducers,
     nodeReducer: nodeReducers,
-    walletsReducer: walletsReducers,
-    pluginsReducer: pluginsReducers
+    walletsReducer: walletsReducers
   };
+  // pluginReducers is a little different
+  // so we don't save them in the reducersDecorators object
+  pluginReducers = [];
+
+  persistReducerWhiteList = [] = [];
```

Comment:
Any reason for the double assignment?

Reply:
Lol none at all. No idea how that got in there.